docs/organizations/organization-overview.md

```diff
@@ -28,6 +28,11 @@ On the Organization Overview you have three areas to help you monitor your repos
 {% if config.extra.show_beta %}
 Use the drop-down list at the top of the page to filter the information displayed on all dashboard areas based on the repositories that you select. For example, you can use the filter to monitor the quality of the repositories maintained by specific teams, or to ignore legacy repositories that are no longer maintained.
 
+!!! tip
+    If you have many repositories, use the language filter to help you narrow down the list of repositories in the drop-down list:
```

Comment:
@prcr Do you think is necessary to say "If you have many repositories," ? That is one scenario. It could be they don't remember the name of the repo but know the languages.  
I suggest to remove that and just start with "Use the languages filter... "

Reply:
Good point @RaquelHipolito! I'm going to make the change.